joomla/joomla-cms
# search filter in getListQuery in ActionlogsModelActionlogs uses the wrong substring when searching by item_id:

Pull Request for Issue # .

### Summary of Changes

search filter in getListQuery in ActionlogsModelActionlogs uses the wrong substring when searching by item_id: - it is set to remove the first 3 chars (as per 'id:') instead of 8 (as per 'item_id:')

### Testing Instructions

Make sure ActionLogs are enabled for com_content and that action logs system plugin is enabled.

Edit and save an article.  View the action logs and you should see something like this

> User admin updated the article Create Article Test 

Hover over the article link to see the article id (in my case it is id=4) - then put it in the text search vox as

> item_id:4

### Expected result

Should see list of actions related to the article 'Create Article Test' (and possibly other actions if you don't also filter by extension ;) )

### Actual result

You get no results since the existing code converts the string "m_id:4" to the integer zero and gets no matching results

### Documentation Changes Required

None
